ingestion/functions/parsing/peru/peru.py

```diff
@@ -1,7 +1,9 @@
+import json
 import os
 import sys
 from datetime import datetime
 import csv
+import json
```

Comment:
nit: another duplicate json import. I'm guessing adding import json to many of these was scripted, which would make sense.